src/main/webui/src/app/components/content/common/StoreListingWidget.jsx

```diff
@@ -49,6 +49,42 @@ CapabilitiesSection.propTypes = {
   store: PropTypes.object
 };
 
+const ConstituentsSection = ({ store }) => {
+  const [toggle, setToggle] = useState(store.display === true);
+
+  store.display = toggle;
+
+  return <div className="right-half">
+    <div className="inline-label">
+      {store.constituents.length} Constituent(s) [
+      <span className="option">
+        <a href="" onClick={e => {
+          e.preventDefault();
+          setToggle(!toggle);
+        }}>{store.display === true ? '-' : '+'}</a>
+      </span>
+      ]
+    </div>
+    {
+      store.display === true &&
+      <ol className="content-panel item-expanded subsection">
+        {
+          store.constituents.map(item => <li key={`constituent-${item}`} className="detail-value-list-item">
```

Comment:
This "detail-value-list-item" class is not present anymore, so we can remove it.

Reply:
removed